TypeCobol.Analysis.Test/BasicCfgInstrs/PerformProcIterativeAfterRecursive1.dot

```diff
@@ -0,0 +1,78 @@
+﻿digraph Cfg {
+compound=true;
+node [
+shape = "record"
+]
+
+edge [
+arrowtail = "empty"
+]
+Block0 [
+label = "{START|}"
+]
+Block1 [
+label = "{Block1|    perform pararec varying n from 1 by 1 until n \> 3\l}"
+]
+subgraph cluster_1{
+color = blue;
+label = "pararec";
+Block10 [
+label = "{PARAREC. Block10|    add 1 to n\l    display n\l    if n \< 3\l}"
+]
+Block11 [
+label = "{Block11|       display 'before perform'\l}"
+]
+Block21 [
+label = "{PARAREC. Block21|    add 1 to n\l    display n\l    if n \< 3\l}"
+]
+Block22 [
+label = "{Block22|       display 'before perform'\l}"
+]
+Block24 [
+label = "{Block24|    else\l       display 'not done'\l}"
+]
+Block25 [
+label = "{Block25|    display 'terminate'\l}"
+]
+Block12 [
+label = "{Block12|       perform pararec test after varying n from 1 by 1 until n\>5\l}"
+]
+subgraph cluster_2{
+color = blue;
+label = "pararec";
+
+}
+Block12 -> Block21 [style=dashed]
+
+Block13 [
+label = "{Block13|       display 'after perform'\l}"
+]
+Block15 [
+label = "{Block15|    display 'terminate'\l}"
+]
+Block14 [
+label = "{Block14|    else\l       display 'not done'\l}"
+]
+Block10 -> Block11
+Block10 -> Block14
+Block11 -> Block21
```

Comment:
Following the same logic as @smedilol's remarks, we should have block11->block12. So, the instruction TEST AFTER should not affect the end graph.